CondCore/PopCon/interface/OnlinePopCon.h

```diff
@@ -0,0 +1,113 @@
+#ifndef CONDCORE_POPCON_ONLINEPOPCON_H
+#define CONDCORE_POPCON_ONLINEPOPCON_H
+
+//
+// Authors:
+//  - Francesco Brivio (Milano-Bicocca)
+//  - Jan Chyczynski   (AGH University of Krakow)
+//
+
+#include "CondCore/DBOutputService/interface/OnlineDBOutputService.h"
+#include "CondCore/CondDB/interface/Time.h"
+#include "CondCore/PopCon/interface/Exception.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
```

Comment:
Not needed here

Reply:
`OnlineDBOutputService.h` and `Exception.h` are needed but I've removed the rest.